server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -173,7 +178,12 @@
             AliasOrIndex aliasOrIndex = metaData.getAliasAndIndexLookup().get(expression);
             if (aliasOrIndex == null ) {
                 if (failNoIndices) {
-                    IndexNotFoundException infe = new IndexNotFoundException(expression);
+                    IndexNotFoundException infe;
+                    if(expression.equals(MetaData.ALL)) {
+                        infe = new IndexNotFoundException("no indices exist", expression);
+                    }else {
```

Comment:
could you add a whitespace between the curly bracket and `else`?

Reply:
Done @javanna 
[IndexNameExpressionResolver](https://github.com/Eunovo/elasticsearch/blob/bf4464a104539ccbbc5102a59751887733f505ab/server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java#L182-L186)